datadog_checks_downloader/datadog_checks/downloader/download.py

```diff
@@ -317,6 +317,11 @@ def get_wheel_relpath(self, standard_distribution_name, version=None, ignore_pyt
             raise MissingVersions(standard_distribution_name)
 
         if not version:
+            # Go through all wheels and remove alphas, betas and rcs
+            non_official_suffixes = ["a", "b", "rc"]
+            for key in list(wheels.keys()):
+                if any(suffix in key for suffix in non_official_suffixes):
```

Comment:
nit: Is the format of the versions of the wheels validated before this point? From my limited understanding of how the versions are named (d.d.d[suffix][num]) this _seems_ like it would never filter an official version of a wheel (always d.d.d?), and I have no idea how an improperly named wheel could be distributed (if at all). However, it could potentially be more defensive to check for \b[d.d.d]\b instead of a, b, or rc in a whole key string (imagine, for example, new non-official suffixes are introduced or a wheel with version 'official' is released).

Reply:
Hello! Thanks for the feedback. I took a look at the tool we use to release (ddev) and there is some validations there:
https://github.com/DataDog/integrations-core/blob/master/datadog_checks_dev/datadog_checks/dev/tooling/commands/release/make.py#L25

it seems to take into account the `d.d.d` format and only allow for suffixes for `rc|pre|alpha|beta`. Those can be all readily caught using the Version package `is_prelease()` function that @fridex mentioned.